prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -29,14 +29,27 @@ import kotlin.random.Random
 /**
  * Azure Functions with HTTP Trigger. Write to blob.
  */
-const val dataRetentionDays = 7L
 const val send = "send"
-const val maxRetryCount = 4
```

Comment:
This count should stay in sync with the size of the array. No idea why we didn't just use `retryDurationInMin.size - 1`? Fixed below, so removed constant.

Reply:
thanks